packages/x-adapter/src/__tests__/endpoint-adapter.factory.spec.ts

```diff
@@ -1,8 +1,8 @@
-import { endpointAdapterFactory } from '../endpoint-adapter.factory';
+import { endpointAdapterFactory } from '../endpoint-adatpter/endpoint-adapter.factory';
```

Comment:
I think we should move this file to the `endpoint-adapter/__tests__` folder.

Reply:
done